db/views/analytics_hmis_case_notes_v01.sql

```diff
@@ -0,0 +1,17 @@
+-- rename from custom_case_notes to hmis_case_notes
+SELECT
+    id,
+    "CustomCaseNoteID" AS custom_case_note_id,
```

Comment:
tbh don't think we need this, did you have an idea for why to keep it?

If we do keep it, maybe rename to `case_note_id` since we dropped the "custom" from the table.

Reply:
I agree, I'll exclude it from export